packages/widgetbook/lib/src/fields/list_field.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter/widgets.dart';
+import 'package:go_router/go_router.dart';
+import 'package:widgetbook/src/fields/field_codec.dart';
+import 'package:widgetbook/src/routing/router.dart';
+import 'package:widgetbook_core/widgetbook_core.dart';
+
+import 'field.dart';
+import 'field_type.dart';
+
+class ListField<T> extends Field<T> {
+  final List<T> values;
+  final String Function(T value) labelBuilder;
+
+  ListField({
```

Comment:
Maybe there is a good reason for the name but I think since this looks and is implemented like a dropdown, we should call it dropdown field. What do you think?

Reply:
I agree that `DropdownField` could be a better name, I think your idea of naming fields after widgets not types will be more reasonable when we have something like `SliderField`.
Will update the PR with the new name 👍